Configuration/HLT/python/autoCondHLT.py

```diff
@@ -1,5 +1,7 @@
 # https://cms-conddb.cern.ch/browser/#search
 
+snapshotTime = "9999-12-31 23:59:59.000"
```

Comment:
implemented in https://github.com/diguida/cmssw/commit/282d9e4f3fc8e55e00bc09c71db4f5e1bd8a0a26